src/main/resources/styles/reporting.css

```diff
@@ -57,6 +57,10 @@
 	background-color: #C5D88A;
 }
 
+.hidden {
+	visibility: hidden;
```

Comment:
That sounds clear but I can't see in this PR where this class is enabled or disabled. Is it new class or re-declared from existing one?